api/reference/conversion_hosts.adoc

```diff
@@ -0,0 +1,181 @@
+[[conversion-hosts]]
+== Conversion Hosts
+
+Management of conversion hosts is provided via the following collection:
+
+[source,data]
+----
+/api/conversion_hosts
+----
+
+The following actions are available for conversion hosts:
+
+* link:#querying-conversion-hosts[Querying conversion hosts]
+* link:#creating-conversion-hosts[Creating conversion hosts]
+* link:#deleting-conversion-hosts[Deleting conversion hosts]
+
+[[querying-conversion-hosts]]
+=== Querying Conversion Hosts
+
+To query all conversion hosts:
+
+----
+GET /api/conversion_hosts
+----
+
+To query a specific conversion host's details:
+
+----
+GET /api/conversion_hosts/:id
+----
+
+[[creating-conversion-hosts]]
+=== Creation Conversion Hosts
+
+Conversion hosts can be created via the _create_ POST action by posting the
+request directly to */api/conversion_hosts*.
+
+[source,json]
+----
+{
+  "resource_type": "ManageIQ::Providers::Redhat::InfraManager::Host",
+  "resource_id": 00000000000004,
+  "auth_user": "root",
+  "conversion_host_ssh_private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvQI[...]gaS2cBrzxn3IQKTabc=\n-----END PRIVATE KEY-----\n",
+  "vmware_vddk_package_url": "http://file.example.com/vddk/VMware-vix-disklib-stable.tar.gz"
+}
+----
+
+==== Conversion Host Attributes
+
+[cols="1<,1<,3<",options="header",]
+|=====================
+| Attribute Group                 | Type    | Description
+| resource_type                   | string a| Type of resource that is acting as a conversion host:
+
+* oVirt / RHV Host: `ManageIQ::Providers::Redhat::InfraManager::Host`
+* oVirt / RHV VM: `ManageIQ::Providers::Redhat::InfraManager::Vm`
+* OpenStack VM: `ManageIQ::Providers::Openstack::CloudManager::Vm`
+
+| resource_id                     | integer | ID of the resource acting as a conversion host
+| auth_user                       | string  | User name for the SSH connection to a conversion host
+| conversion_host_ssh_private_key | string  | SSH private key for SSH connection to a conversion host. Line breaks must be replaced with `\n`.
+
+| vmware_vddk_package_url         | string  | *_Only for VDDK transport_*. URL of downloaded VMware VDDK `tar.gz` archive file. The VDDK must be saved in an HTML-accessible location.
+
+| vmware_ssh_private_key          | string  | *_Only for SSH transport_*. SSH private key for SSH connection to the VMware ESXi hosts. Line breaks must be replaced with `\n`.
+
+| openstack_tls_ca_certs          | string  | *_Only for OpenStack conversion host_*. CA certificate chain for validating the OpenStack API endpoint certificate.
```

Comment:
In the meantime, we have changed the API, so it should be:

```
| tls_ca_certs          | string  | CA certificate chain for validating the provider API endpoint certificate.
```

Reply:
Done